agent/config/deep-copy.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
```

Comment:
This need to be removed for backports

Reply:
Apologies @dhiaayachi I missed this. This has been done 🙇